packages/klevu-ui/src/components/klevu-color-swatch/klevu-color-swatch.css

```diff
@@ -1,14 +1,24 @@
 :host {
   display: block;
+  --klevu-color-swatch-border: #8c8c8c;
+  --klevu-color-swatch-default-background: #000;
+  --klevu-color-swatch-size: 25px;
+  --klevu-color-swatch-selected-color: #2b4af7;
+  --klevu-color-swatch-border-radius: 50%;
 }
 
 .circle {
   margin: 3px;
```

Comment:
There shouldn't be any margin in the component itself. It should be handled by the container always.

So for in example I would to `klevu-facet` a container that has `display: flex; gap: var(--klevu-spacing-02)` that would make the sizing good.

Reply:
addressed